### ISSUE TITLE
Class `MyClass` must implement all abstract methods of `Model`

### ISSUE THREAD

**Issue by Olegt0rr** [art049/odmantic#68]
## Description
Trying to implement odmantic (0.3.1), but get python hints:
```
Class Job must implement all abstract methods
```

## My code
```python
from odmantic import Model

class Job(Model):
    created: datetime
    state: str
```
## Screenshots

![image](https://user-images.githubusercontent.com/25399456/101894937-edd2c800-3bb7-11eb-98d6-80a4578d50aa.png)

Alse when I'm trying to add ABC methods I receive this one:

![image](https://user-images.githubusercontent.com/25399456/101894967-f62b0300-3bb7-11eb-9cea-2cda8f2396f5.png)

## Summary
If this action is not nesessary, please remove abstract method mark.
If this action is nesessary, please, add it to examples

**Comment by art049** [comment]
Hmm, i think this error appears because copy is currently raising a non implemented error.
It should be fixed by #39 .

**Comment by Olegt0rr** [comment]
> Hmm, i think this error appears because copy is currently raising a non implemented error.
> It should be fixed by #39 .

It's related, but this is not the reason :)
PyCharm knows that with "Pythonic way" every abrastract method should be overrided by child class.
I've found only one way to fix it - add `copy` method to `Model`.
But I still believe that another way exists :)

**Comment by art049** [comment]
Thanks :smile: , I will have a look with Pycharm before merging #39 to fix this issue.
